controller/config_test.go

```diff
@@ -122,26 +122,40 @@ var validateTests = []struct {
 	},
 	expectError: `invalid identity public key: wrong length for base64 key, got 3 want 32`,
 }, {
-	about: "invalid management space name",
+	about: "invalid management space name - whitespace",
 	config: controller.Config{
 		controller.CACertKey:           testing.CACert,
-		controller.JujuManagementSpace: ` `,
+		controller.JujuManagementSpace: " ",
 	},
-	expectError: `invalid juju mgmt space name`,
+	expectError: `juju mgmt space name   not valid`,
```

Comment:
mostly I just noticed that this error is completely not understandable, but would be with %q